docs/understand/enterprise/orchestration.md

```diff
@@ -0,0 +1,389 @@
+      - LAKEFS_AUTH_ENCRYPT_SECRET_KEY="some random secret string"
+      - LAKEFS_AUTH_API_ENDPOINT=http://fluffy:9000/api/v1
+      - LAKEFS_AUTH_API_SUPPORTS_INVITES=true
+      - LAKEFS_AUTH_LOGOUT_REDIRECT_URL=http://fluffy:8000/oidc/logout
+      - LAKEFS_AUTH_UI_CONFIG_LOGIN_URL=http://fluffy:8000/oidc/login
+      - LAKEFS_AUTH_UI_CONFIG_LOGOUT_URL=http://fluffy:8000/oidc/logout
+      - LAKEFS_AUTH_UI_CONFIG_RBAC=internal
+      - LAKEFS_AUTH_UI_CONFIG_LOGIN_COOKIE_NAMES=[internal_auth_session,oidc_auth_session]
+      - LAKEFS_AUTH_OIDC_FRIENDLY_NAME_CLAIM_NAME="nickname"
+      - LAKEFS_AUTH_OIDC_DEFAULT_INITIAL_GROUPS=["Admins"]
+      - LAKEFS_AUTH_AUTHENTICATION_API_ENDPOINT=http://fluffy:8000/api/v1
+      - LAKEFS_AUTH_AUTHENTICATION_API_EXTERNAL_PRINCIPALS_ENABLED=true
+      - LAKEFS_DATABASE_TYPE=postgres
+      - LAKEFS_DATABASE_POSTGRES_CONNECTION_STRING=postgres://lakefs:lakefs@postgres/postgres?sslmode=disable
+      - LAKEFS_BLOCKSTORE_TYPE=local
+      - LAKEFS_BLOCKSTORE_LOCAL_PATH=/home/lakefs
+      - LAKEFS_BLOCKSTORE_LOCAL_IMPORT_ENABLED=true
+    entrypoint: ["/app/wait-for", "postgres:5432", "--", "/app/lakefs", "run"]
+
+  postgres:
+    image: "postgres:11"
+    ports:
+      - "5433:5432"
+    environment:
+      POSTGRES_USER: lakefs
+      POSTGRES_PASSWORD: lakefs
+
+  fluffy:
+    image: "${FLUFFY_REPO:-treeverse}/fluffy:${TAG:-0.4.0}"
+    command: "${COMMAND:-run}"
+    ports:
+      - "8000:8000"
+      - "9000:9000"
+    depends_on:
+      - "postgres"
+    environment:
+      - FLUFFY_LOGGING_LEVEL=DEBUG
+      - FLUFFY_DATABASE_TYPE=postgres
+      - FLUFFY_DATABASE_POSTGRES_CONNECTION_STRING=postgres://lakefs:lakefs@postgres/postgres?sslmode=disable
+      - FLUFFY_AUTH_ENCRYPT_SECRET_KEY="some random secret string"
+      - FLUFFY_AUTH_SERVE_LISTEN_ADDRESS=0.0.0.0:9000
+      - FLUFFY_LISTEN_ADDRESS=0.0.0.0:8000
+      - FLUFFY_AUTH_SERVE_DISABLE_AUTHENTICATION=true
+      - FLUFFY_AUTH_LOGOUT_REDIRECT_URL=<oidc-login-url>
+      - FLUFFY_AUTH_POST_LOGIN_REDIRECT_URL=http://lakefs:8000/
+      - FLUFFY_AUTH_OIDC_ENABLED=true
+      - FLUFFY_AUTH_OIDC_URL=<oidc-endpoint>
+      - FLUFFY_AUTH_OIDC_CLIENT_ID=<client-id>
+      - FLUFFY_AUTH_OIDC_CLIENT_SECRET=<client-secret>
+      - FLUFFY_AUTH_OIDC_CALLBACK_BASE_URL=http://fluffy:8000
+      - FLUFFY_AUTH_OIDC_IS_DEFAULT_LOGIN=true
+      - FLUFFY_AUTH_OIDC_LOGOUT_CLIENT_ID_QUERY_PARAMETER=client_id
+    entrypoint: [ "/app/wait-for", "postgres:5432", "--", "/app/fluffy" ]
+    configs:
+      - source: fluffy.yaml
+        target: /etc/fluffy/config.yaml
+
+ #This tweak is unfortunate but also necessary. logout_endpoint_query_parameters is a list
+ #of strings which isn't parsed nicely as env vars.
+configs:
+  fluffy.yaml:
+    content: |
+      auth:
+        oidc:
+          logout_endpoint_query_parameters:
+            - returnTo
+            - http://localhost:8080/oidc/login
+```
+
+## More values file examples
```

Comment:
The title is confusing or the section itself.
One tab shows Helm values (LDAP) while the second (ADFS) shows regular configuration parameters (non helm). 
Either split the section or be consistent with the example context. (Helm vs non helm - i think non helm is better).

Reply:
Separated them, let me know what you think